pytorch_cnn_trainer/engine.py

```diff
@@ -347,8 +362,8 @@ def fit(
                 break
 
         print("Done Training, Model Saved to Disk")
-
-    return True  # For now, we should probably return an history object like keras.
+    history = {"train":train_metrics, "val" : valid_metrics}
```

Comment:
What `train_metrics` are coming here?
If you see already metrics per epoch were being returned. It would be wiser to edit only `fit()` function to get metrics logged.

I think you misunderstood the idea.

Idea was only edit `fit()` which already has metrics, and they could be simply logged in history dictionary.

Reply:
ok then instead all stuff I did basically I have to add the following line in fit "history = {"train":train_metrics, "val" : valid_metrics}" right